src/uu/ls/src/ls.rs

```diff
@@ -283,6 +295,21 @@ fn sort_entries(entries: &mut Vec<PathBuf>, options: &getopts::Matches) {
     }
 }
 
+#[cfg(windows)]
+fn should_display(entry: &DirEntry, options: &getopts::Matches) -> bool {
+    let ffi_name = entry.file_name();
+    let name = ffi_name.to_string_lossy();
+    let hidden_by_ntfs = is_hidden(&entry.path()).unwrap();
+    if !options.opt_present("a") && !options.opt_present("A") && name.starts_with('.') || hidden_by_ntfs {
```

Comment:
Seems that you are duplicating a bunch of code. Would it be possible to factorize this code?

Reply:
I tried to do that, but it says that cfg notations aren't available for if statements